src/redirect.cc

```diff
@@ -81,6 +81,8 @@ redirectHandleReply(void *data, const Helper::Reply &reply)
 {
     RedirectStateData *r = static_cast<RedirectStateData *>(data);
     debugs(61, 5, "reply=" << reply);
+    if (reply.expires.has_value())
+        debugs(61, 5, "ignoring unexpected ttl=" << (reply.expires.value() - current_time.tv_sec));
```

Comment:
If possible, please enhance/configure the right `helper` class objects to honor TTLs (and warn when they are supplied but not honored) instead of hoping that we will remember to copy code snippets like this to every new helper response handler that does _not_ support TTLs.